src/main/resources/com/hermesworld/ais/galapagos/topics/service/impl/messages_de.properties

```diff
@@ -0,0 +1,30 @@
+#TopicServiceImpl class keys
+UNKNOWN_APPLICATION_ID=Unbekannte Anwendung ID.
+APPLICATION_NOT_REGISTERED_ON_ENVIRONMENT=Awendung {0} ist nicht auf der Umgebung {1} registriert.
+CANNOT_PARSE_JSON_SCHEMA=Konnte JSON-Schema nicht parsen.
+CURRENT_USER_IS_NO_OWNER_OF_APPLICATION=Der aktuelle Benutzer ist kein Besitzer der Anwendung {0}.
+FOR_COMMAND_SUBSCRIBE_TOPIC_ADD_PRODUCER=F�r den Befehl Topics abonnieren Sie das Topic, um einen neuen Producer hinzuf�gen.
+FOR_COMMAND_SUBSCRIBE_TOPIC_REMOVE_PRODUCER=F�r den Befehl Topics abonnieren Sie das Topic, um einen neuen Producer zu entfernen.
+CANNOT_CHANGE_OWNER_INTERNAL_TOPICS=Kann den Besitzer f�r interne Topics nicht �ndern.
+CANNOT_UPDATE_SUBSCRIPTION_APPROVAL_REQUIRED_INTERNAL_FLAG_TOPIC=Kann den subscriptionAPPROVALRequired-Flag f�r interne Topics der Anwendung nicht aktualisieren.
+NO_USER_CURRENTLY_LOGGED_IN=Kein Benutzer ist derzeit angemeldet.
+NO_SCHEMA_CURRENT_STAGE_TOPIC=Kein Schema auf der aktuellen Stufe f�r das Topic {0}.
+SCHEMA_ALREADY_STAGED=Das ausgew�hlte Schema existiert bereits auf der n�chsten Stufe. Um dieses Schema zu l�schen,\ m�ssen Sie es dort zuerst l�schen!\.
+CANNOT_ADD_JSON_SCHEMAS_INTERNAL_TOPICS=JSON-Schemas k�nnen nicht zu internen Topics hinzugef�gt werden.
+JSON_SCHEMA_MUST_DECLARE_SCHEMA_VALUE=JSON-Schema muss den Wert \"$schema\" auf der oberesten Ebene deklarieren.
```

Comment:
Backslashes not required here, I think (otherwise, they would also be required in the next line)